test.js

```diff
@@ -56,7 +56,7 @@ global.Runners = {
       fingerroot500: 1,
       arrowhead600: 1,
       arrowhead631: 1,
-      lovell700: -1,
+      lovell700: 1,
```

Comment:
I thought we planned on staring with Lovell inactive. Then test pre and post Lovell